docs/get-started/gpu_dataproc_packages_ubuntu_sample.sh

```diff
@@ -26,7 +26,7 @@ readonly CUDA_VERSION
 readonly NVIDIA_BASE_DL_URL='https://developer.download.nvidia.com/compute'
 # Parameters for NVIDIA-provided Ubuntu GPU driver
 readonly NVIDIA_UBUNTU_REPOSITORY_URL="${NVIDIA_BASE_DL_URL}/cuda/repos/ubuntu1804/x86_64"
-readonly NVIDIA_UBUNTU_REPOSITORY_KEY="${NVIDIA_UBUNTU_REPOSITORY_URL}/7fa2af80.pub"
+readonly NVIDIA_UBUNTU_REPOSITORY_KEY="${NVIDIA_UBUNTU_REPOSITORY_URL}/3bf863cc.pub"
```

Comment:
Should we update our instructions per https://forums.developer.nvidia.com/t/notice-cuda-linux-repository-key-rotation/212772#install-new-cuda-keyring-package-3 , since it appears that manually installing a new signing key is not preferred?

Reply:
Yes we can do that. Let me firstly submit https://github.com/GoogleCloudDataproc/initialization-actions/pull/995 to change in GCP repo for this.

Let me move this PR to draft firstly.

Reply:
@sameerz @jlowe  I just updated this PR:
1. Use latest CUDA 11.6 on Ubuntu 20.04 instructions in the doc since we want users to use latest CUDA for on-prem installation.
2. Changed `NVIDIA_UBUNTU_REPOSITORY` to `NVIDIA_UBUNTU_REPO` to match GCP [PR-991](https://github.com/GoogleCloudDataproc/initialization-actions/pull/991/).
3. Use the key package way to install key.